### PR TITLE
Remove PThreads note about SAB removal

### PR DESCRIPTION
**Pull request by RReverser**
That note is tecnically outdated now and seems confusing alongside the new note above. Browsers now did start re-enabling SAB, and using it in Chrome without flags is no longer allowed either.

### CODE REVIEW

**Review by sbc100 — APPROVED** [review]

**Comment by brion** [comment]
Safari has not shipped SharedArrayBuffer yet, so it's not accurate to say that "browsers" as a whole are shipping it being COEP/CORP.

**Comment by RReverser** [comment]
> "browsers" as a whole are shipping it behind COEP/CORP

I think it's meant to be read as "have started shipping" not "all of them" but I can see how it can be confusing, hence added a link to the support table as well.

**Comment by RReverser** [comment]
Let me rephrase to "Some browsers" or something similar...

**Comment by RReverser** [comment]
@brion Reworded a bit, how does that sound?

**Comment by brion** [comment]
> @brion Reworded a bit, how does that sound?

Perfect, thanks :)

-- signed, the one poor guy who deploys WebAssembly specifically to Safari ;)
